src/DataCollector/ViewCollector.php

```diff
@@ -44,6 +70,36 @@ public function getWidgets()
         ];
     }
 
+    /**
+     * Get the editor href for a given file and line, if available.
+     *
+     * @param string $filePath
+     * @param int    $line
+     *
+     * @return null|string
+     * @throws InvalidArgumentException If editor resolver does not return a string
+     *
+     */
+    protected function getEditorLink($filePath)
+    {
+        if (empty(config('debugbar.editor'))) {
+            return null;
+        }
+
+        if (empty($this->editors[ config('debugbar.editor') ])) {
+            throw new InvalidArgumentException(
+                'Unknown editor identifier: ' . config('debugbar.editor') . '. Known editors:' .
+                implode(', ', array_keys($this->editors))
+            );
+        }
+
+        $filePath = $this->replaceSitesPath($filePath);
+
+        $url = str_replace(['%file'], [$filePath], $this->editors[ config('debugbar.editor') ]);
+
+        return $url;
+    }
+
```

Comment:
This function checks if the IDE's URL is available or not, and if it is, it returns the full url of the file in the IDE by merging the editor link and the file link together